modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -54,15 +55,16 @@ exports.update = function (req, res) {
 exports.changeProfilePicture = function (req, res) {
   var user = req.user;
   var message = null;
+  var upload = multer({ dest: './modules/users/client/img/profile/uploads/' }).single('newProfilePicture');
```

Comment:
why did we remove folder configuration from the more generic area in express (https://github.com/meanjs/mean/pull/948/files#diff-16e0a4c137ae50cf68b1d2c3c914be85L93) to the controller?

It makes sense the an upload folder is a configuration item and someone should be able to configure it, even actually to take it out from express and put it as part of the env configuration, but definitely not to hard code it in the controller.